conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -9,11 +9,6 @@ environment:
   # See: http://stackoverflow.com/a/13751649/163740
   CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
 
-  # We set a default Python version for the miniconda that is to be installed. This can be
-  # overridden in the matrix definition where appropriate.
-  CONDA_PY: "27"
```

Comment:
Opened this PR ( https://github.com/conda-forge/conda-smithy/pull/392 ) to suggest dropping it if possible. Will come up with a workaround in the interim while that is discussed.